creator-node/src/routes/tracks.js

```diff
@@ -19,46 +19,43 @@ module.exports = function (app) {
   app.post('/track_content', authMiddleware, ensurePrimaryMiddleware, syncLockMiddleware, trackFileUpload.single('file'), handleResponse(async (req, res) => {
     if (req.fileFilterError) return errorResponseBadRequest(req.fileFilterError)
     const routeTimeStart = Date.now()
+    let codeBlockTimeStart = Date.now()
 
     // create and save track file segments to disk
     let segmentFilePaths
     try {
       req.logger.info(`Segmenting file ${req.fileName}...`)
-      const segmentTimeStart = Date.now()
       segmentFilePaths = await ffmpeg.segmentFile(req, req.fileDir, req.fileName)
-      req.logger.info(`Time taken to segment track file: ${Date.now() - segmentTimeStart}ms for file ${req.fileName}`)
+      req.logger.info(`Time taken in /track_content to segment track file: ${Date.now() - codeBlockTimeStart}ms for file ${req.fileName}`)
```

Comment:
@hareeshnagaraj ditto with these logs. i feel like we know ffmpeg is the slowest thing now

Reply:
yes